packages/app/src/components/pages/VerifyEmail/index.js

```diff
@@ -1,11 +1,19 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import utilsAPI from '../../../utils/API/nominationsAPI.js';
+
+const VerifyEmail = (props) => {
+  
+    const token = useState(props.match.params.token)
```

Comment:
I don't think you need `useState` here, as it's generally meant for things that change state within the page. In this case, the token is going to be determined by the URL, and we won't modify it after the page loads.

Reply:
Yeah thats what I thought too but wasn't sure after reading this, 
https://reactjs.org/docs/hooks-state.html
"Normally, variables “disappear” when the function exits but state variables are preserved by React."
and since useEffect is called after rendering the html, I wasn't sure.  Perhaps the quote refers to updating only??

Reply:
no problem, will change it

Reply:
The "disappearing" refers to variables that rely on some internal state. If I increment a counter every time someone clicks a button, that counter will lose its state if the component re-renders unless the state is stored using `useState` or something similar. In this case, the value depends only on what's passed in from props which is external to the function so it doesn't need to be kept in a state variable.